### ISSUE TITLE
Exclude by default instead of Include (whitelist instead of blacklist)

### ISSUE THREAD

**Issue by dorongutman** [SumoLogic/fluentd-kubernetes-sumologic#81]
Can you add a configuration that will exclude everything that doesn't have an annotation that specifically declare that resource as needs sumologic handling ? 
So basically, instead of sending logs from ALL resource excluding specific resources that have an appropriate annotation, don't send logs at all, except for specific resources which I include using an annotation.

**Comment by frankreno** [comment]
@dorongutman were you able to try to the above to achieve the desired behavior?

**Comment by dorongutman** [comment]
@frankreno I installed the helm chart with `--set sumologic.excludePodRegex=".*"` and while pod logs are not showing in sumologic (which is good), I see the following in sumologic, about 4 times per minute:
```
{"timestamp":1540830720057,"message":"stats - namespace_cache_size: 2, pod_cache_size: 10, namespace_cache_api_updates: 15, pod_cache_api_updates: 15, id_cache_miss: 15"}
```

For now, I don't want to see ANY logs in sumologic from kubernetes, except from the pods I specifically said I want to get the application logs from.

**Comment by frankreno** [comment]
Ok good news is that I know what's happening and why those logs are showing up.  Will put in a fix soon for that after I fix some other issues. In the short term, you can exclude this by using a processing rule on the HTTP source. https://help.sumologic.com/Manage/Collection/Processing-Rules/Include-and-Exclude-Rules

**Comment by frankreno** [comment]
This has been fixed in v2.0.0
